config/routes.rb

```diff
@@ -3,4 +3,8 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+  root "users#index"
+  resources :users, only: [:index, :show]
+  get '/users/:id/posts', to: 'posts#index', as: 'user_posts'
+  get '/users/:id/posts/:id', to: 'posts#show', as: 'user_post'
```

Comment:
- [ ] I would suggest replacing the routes code with the below code. It will make the code more clean and professional.

```ruby
resources :users, only: [:index, :show] do
    resources :posts, only: [:index, :show]
  end